Cannot compile plugin agains latest coredns

```
CGO_ENABLED=0  go build -v -ldflags="-s -w -X github.com/coredns/coredns/coremain.GitCommit=f6262eb2-dirty" -o coredns
github.com/c-mueller/ads
# github.com/c-mueller/ads
/go/pkg/mod/github.com/c-mueller/ads@v0.2.2/setup.go:57:4: undefined: metrics.MustRegister
/go/pkg/mod/github.com/c-mueller/ads@v0.2.2/setup.go:58:4: undefined: metrics.MustRegister
/go/pkg/mod/github.com/c-mueller/ads@v0.2.2/setup.go:59:4: undefined: metrics.MustRegister
/go/pkg/mod/github.com/c-mueller/ads@v0.2.2/setup.go:60:4: undefined: metrics.MustRegister
make: *** [Makefile:16: coredns] Error 2
```

--- a/go.mod
+++ b/go.mod
@@ -1,21 +1,11 @@
 module github.com/c-mueller/ads
 
-go 1.13
+go 1.15
 
 require (
-	github.com/Flaque/filet v0.0.0-20190209224823-fc4d33cfcf93
-	github.com/caddyserver/caddy v1.0.5
-	github.com/coredns/coredns v1.7.0
+	github.com/coredns/caddy v1.1.0 // indirect
+	github.com/coredns/coredns v1.7.1-0.20200828130837-ecd61a21656a
 	github.com/miekg/dns v1.1.31
-	github.com/opentracing/opentracing-go v1.2.0 // indirect
 	github.com/prometheus/client_golang v1.7.1
-	github.com/prometheus/common v0.13.0 // indirect
-	github.com/stretchr/testify v1.5.1
-	golang.org/x/crypto v0.0.0-20200820211705-5c72a883971a // indirect
-	golang.org/x/net v0.0.0-20200822124328-c89045814202
-	golang.org/x/sys v0.0.0-20200828194041-157a740278f4 // indirect
-	golang.org/x/text v0.3.3 // indirect
-	google.golang.org/genproto v0.0.0-20200828030656-73b5761be4c5 // indirect
-	google.golang.org/grpc v1.31.1 // indirect
-	google.golang.org/protobuf v1.25.0 // indirect
+	golang.org/x/net v0.0.0-20200324143707-d3edc9973b7e
 )

--- a/go.mod
+++ b/go.mod
@@ -1,4 +1,3 @@
-
 module github.com/c-mueller/ads
 
 go 1.13
@@ -6,9 +5,17 @@ go 1.13
 require (
 	github.com/Flaque/filet v0.0.0-20190209224823-fc4d33cfcf93
 	github.com/caddyserver/caddy v1.0.5
-	github.com/coredns/coredns v1.6.7
-	github.com/miekg/dns v1.1.28
-	github.com/prometheus/client_golang v1.5.0
+	github.com/coredns/coredns v1.7.0
+	github.com/miekg/dns v1.1.31
+	github.com/opentracing/opentracing-go v1.2.0 // indirect
+	github.com/prometheus/client_golang v1.7.1
+	github.com/prometheus/common v0.13.0 // indirect
 	github.com/stretchr/testify v1.5.1
-	golang.org/x/net v0.0.0-20200320181208-1c781a10960a
+	golang.org/x/crypto v0.0.0-20200820211705-5c72a883971a // indirect
+	golang.org/x/net v0.0.0-20200822124328-c89045814202
+	golang.org/x/sys v0.0.0-20200828194041-157a740278f4 // indirect
+	golang.org/x/text v0.3.3 // indirect
+	google.golang.org/genproto v0.0.0-20200828030656-73b5761be4c5 // indirect
+	google.golang.org/grpc v1.31.1 // indirect
+	google.golang.org/protobuf v1.25.0 // indirect
 )

--- a/go.mod
+++ b/go.mod
@@ -1,11 +1,11 @@
 module github.com/c-mueller/ads
 
-go 1.15
+go 1.13
 
 require (
-	github.com/coredns/caddy v1.1.0 // indirect
-	github.com/coredns/coredns v1.7.1-0.20200828130837-ecd61a21656a
+	github.com/caddyserver/caddy v1.0.5
+	github.com/coredns/coredns v1.7.1
 	github.com/miekg/dns v1.1.31
 	github.com/prometheus/client_golang v1.7.1
-	golang.org/x/net v0.0.0-20200324143707-d3edc9973b7e
+	golang.org/x/net v0.0.0-20200923182212-328152dc79b1
 )

--- a/setup.go
+++ b/setup.go
@@ -20,11 +20,11 @@ import (
 	"github.com/caddyserver/caddy"
 	"github.com/coredns/coredns/core/dnsserver"
 	"github.com/coredns/coredns/plugin"
-	"github.com/coredns/coredns/plugin/metrics"
+	"github.com/prometheus/client_golang/prometheus"
 	"time"
 )
 
-const Version = "0.2.3"
+const Version = "0.2.4"
 
 func init() {
 	caddy.RegisterPlugin("ads", caddy.Plugin{
@@ -56,8 +56,7 @@ func setup(c *caddy.Controller) error {
 
 	c.OnStartup(func() error {
 		once.Do(func() {
-			metrics.MustRegister(c, requestCount)
-			metrics.MustRegister(c, blockedRequestCount)
+			prometheus.MustRegister(requestCount, blockedRequestCount)
 			updater.Start()
 		})
 		return nil

